frontend/src/lib/components/ScriptEditor.svelte

```diff
@@ -262,7 +262,7 @@
 <SplitPanesWrapper>
 	<Splitpanes class="!overflow-visible">
 		<Pane size={60} minSize={10} class="!overflow-visible">
-			<div class="pl-2 h-full !overflow-visible">
```

Comment:
I think there was a good reason for that, as in line number too close otherwise

Reply:
reverted